buildSrc/build.gradle.kts

```diff
@@ -41,8 +42,8 @@ project.tasks.withType(JavaCompile::class.java).configureEach {
 }
 
 // ensure the Kotlin + Java compilers both use the same language level.
-project.tasks.withType(KotlinCompile::class.java).configureEach {
-    kotlinOptions {
-        jvmTarget = JavaVersion.VERSION_1_8.toString()
+kotlin {
+    compilerOptions {
+        jvmTarget.set(JvmTarget.JVM_1_8)
```

Comment:
isn't our min required version Java 11?

Reply:
For building the plugin, yes. For the SDK we still use 1.8. I'll bump this in a separate PR